frontend/scss/components/organisms/toc.scss

```diff
@@ -48,13 +48,14 @@
     position: sticky;
     z-index: 1;
     padding: 0 10px 0 0;
+    margin: 0;
     height: auto;
-    top: 114px;
+    top: 90px;
     right: auto;
     background: none;
     box-shadow: none;
     overflow-y: scroll;
-    max-height: calc(100vh - 214px);
+    max-height: calc(100vh - 90px);
```

Comment:
nit: I assume the 90px is related to the top value, in that case it'd be good to extract the value into a variable.

Reply:
Thanks for the feedback!
That's right, I also thought about extracting all that spacing calculations into variables, but this seems to me like an own new task.
It depends on individual font-sizes and other metrics.
I'll create a new issue for this.

Reply:
There it is: #2272